### PR TITLE
Final-Capstone-Frontend: Review Request

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this final milestone, the following was done:

- We followed the layout of the provided design except for the titles, descriptions, and photos
- We Built a Navigation component containing the "houses, the reservations, the added house, the reserve and remove house options"
- We added authentication whereby the logged-in user can only delete the house he has added
- We added an authentication page for user logging
- We made the app responsive, creating both mobile and desktop versions.
- We built the details page for each selected house
- We Added the API Documentation
- We use Use Rails to create backend API.
- We Use React & Redux to create frontend UI.
- The project was set up as two separate apps with the backed being [here](https://github.com/andersonlebon/Final-Capstone-backend/tree/development)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,106 @@
...
+## Built With
+- React-Redux
+- HTML/SCSS
+- [Final-capstone-Back-End API](https://house-booking-api.herokuapp.com/api-docs/index.html)
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Live Demo
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Find the application's live version [HERE]()
```
Comment (MuriloRoque) [inline README.md:37]: - [x] Just so you don't forget to update this link 😄

File: src/sass/_reservation.scss [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    width: 93%;
+    height: 80%;
+    background-color: #81a30f80;
+    border-radius: 10px;
+  }
+}
+
+.image-bg {
+  position: absolute;
+  // background-color: #96bf0f11;
```
Comment (MuriloRoque) [inline src/sass/_reservation.scss]: - [x] Remove this commented code.

File: src/sass/_reservation.scss [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    background-color: #364407;
+    opacity: 0.8;
+  }
+}
+
+.react-datepicker {
+  background-color: #b0b895 !important;
+  // .react-datepicker__current-month, .react-datepicker__day-name {
+  //      color: $secondary-color !important;
+  // }
```
Comment (MuriloRoque) [inline src/sass/_reservation.scss]: - [x] Remove this commented code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
Hi @pasytchangwa, @toliboff, @andersonlebon, 

Very good job so far! Your front-end is very good and I think this is mostly ready, there are only some very small details I could catch.

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/sass/_reservation.scss (hunk 2 above)
Comment (pasytchangwa) [reply src/sass/_reservation.scss]: done

File: src/sass/_reservation.scss (hunk 3 above)
Comment (pasytchangwa) [reply src/sass/_reservation.scss]: done

File: README.md (hunk 1 above)
Comment (pasytchangwa) [reply README.md:37]: Thank you for the reminder. We decided to deploy it at the end after the reviews.

**Review by MuriloRoque — APPROVED** [review]
Hi @pasytchangwa, @andersonlebon, @toliboff, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**